app/helpers/application_helper.rb

```diff
@@ -6,4 +6,14 @@ def identifier_types
   def percentage(ratio)
     "#{(ratio * 100).round}%"
   end
+
+  def github_search_enabled?
+    ENV['GITHUB_ORGANIZATION'].present?
+  end
+
+  def github_search_url(split)
+    return nil unless github_search_enabled?
+
+    "https://github.com/search?q=org%3A#{ENV['GITHUB_ORGANIZATION']}+#{split.name}&type=code"
+  end
```

Comment:
I think we should lift some of this code out into a configuration model. It's a bit wonky to have this "feature is enabled" code exist in a view helper IMO. lemme sketch out a way to do that...

Reply:
okay, so here's my refactoring 81a5c539472a8bb593749a2a9e903c94b73039a6